### DISCUSSION TITLE
Please set quasar.config.js > bin > linuxAndroidStudio with the escaped path to your studio.sh

### DISCUSSION THREAD

**Discussion by bobwatcherx** [quasarframework/quasar#14996] [CLI - Capacitor / Cordova modes]
i want debug my project to android devices
but i have problem like this

```
 ⚠️  
 ⚠️  Opening Android Studio IDE...
 ⚠️  From there, use the IDE to run the app.
 ⚠️  
 ⚠️  DO NOT close the terminal as this will kill the devserver.
 ⚠️  
 ⚠️  DO NOT upgrade Gradle or any other deps as Android Studio will suggest.
 ⚠️  If you encounter any IDE errors then click on File > Invalidate caches and restart.
 ⚠️  

 App • ⚠️  Cannot determine path to IDE executable
 Please set quasar.config.js > bin > linuxAndroidStudio with the escaped path to your studio.sh
 Example: '/usr/local/android-studio/bin/studio.sh'
```
when I open the quasar.conf.js file and I add it under the words capacitor like this 
`
// Full list of options: https://v2.quasar.dev/quasar-cli/developing-capacitor-apps/configuring-capacitor 
capacitor: { hideSplashscreen: true, linuxAndroidStudio:"/root/Android/Sdk" },`

and I tried debugging again with the quasar dev -m capacitor -T android command, it still shows an error like that.  please provide a complete explanation.  where should i put the code

**Comment by pdanpdan** [comment]
This (`/root/Android/Sdk`) is the name of your executable `studio.sh`?

**Reply by bobwatcherx** [reply]
> Ini ( `/root/Android/Sdk`) adalah nama file yang dapat dieksekusi `studio.sh`?

root@minto-Lenovo-V145-14AST:/home/minto/belajar/projectpro/project2# echo $ANDROID_HOME
`/root/Android/Sdk`

and here is my capacitor.config.json file
```
{
  "appId": "com.pratama.bangunan",
  "appName": "pratama bagunan",
  "bundledWebRuntime": false,
  "npmClient": "yarn",
  "webDir": "www",
  "linuxAndroidStudioPath": "/home/minto/Downloads/android/android-studio/bin"
}


```
What is wrong .  please give a solution I'm stuck when I want to build an apk

**Reply by bobwatcherx** [reply]
> Ini ( `/root/Android/Sdk`) adalah nama file yang dapat dieksekusi `studio.sh`?

the location of studio.sh on my android studio leads in the folder

`/home/minto/Downloads/android/android-studio/bin`

what do I have to do ?

**Comment by fmndako** [comment]
Add the configuration directly to the root of your `quasar.config.js` object, rather than placing it under the `capacitor` or any other nested property.". 
`module.exports = configure(function (/* ctx */) {
  return {
    // other configs
    bin: {
       linuxAndroidStudio: '/path/to/your/android/android-studio/bin/studio.sh',
    },
    // other configs
  }
});`

Ensure that the path also includes `studio.sh`

**Comment by DarkLite1** [comment]
Had the same issue/question. Maybe this could be more clear in the error description, the placement of the code:

```
import { configure } from 'quasar/wrappers';
import { fileURLToPath } from 'node:url';

export default configure((ctx) => {
  return {
    bin: { 
      linuxAndroidStudio: '/usr/local/bin/android-studio/bin/studio' 
    },
```

**Reply by ociegonzalez** [reply]
Ty so much!
